src/midi/midi_fluidsynth.h

```diff
@@ -23,19 +23,25 @@
 
 #include "midi_device.h"
 
-#if C_FLUIDSYNTH
-
 #include <atomic>
 #include <fluidsynth.h>
 #include <memory>
 #include <optional>
 #include <thread>
 #include <vector>
 
+#include "dynlib.h"
 #include "mixer.h"
 #include "rwqueue.h"
 #include "std_filesystem.h"
 
+namespace fsynth {
```

Comment:
@shermp Can you please change this `namespace FluidSynth`? We use UpperCamelCase for namespaces (e.g. `namespace SoundCanvas`).

Reply:
Done.